Big-Spender/readme.md

```diff
@@ -48,7 +48,8 @@ You are working with Claire and Farnoosh, who are trying to complete a missing r
 **You:** Absolutely. Here's the SQL query you need:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends
```

Comment:
Good, but I'm not a fan of using * in SELECT clauses unless there is a really good reason. Can you tell me why it's not considered best practice?